### PR TITLE
Application Connector CRD desc. update + fixes

### PR DESCRIPTION
**Pull request by tomekpapiernik**
**Description**

Changes proposed in this pull request:

Application Connector CRD desc. update + fixes

### CODE REVIEW

File: docs/application-connector/docs/044-remote-evironment-custom-resource.md [hunk 1]
```diff
@@ -29,9 +29,27 @@ spec:
...
+| **spec.services** |    **NO**   | Contains all services that the Remote Environment provides. |
+| **spec.services.id** |    **YES**   | Identifies the service that the Remote Environment provides. |
+| **spec.services.identifier** |    **NO**   | Provides an additional identifier of the ServiceClass. |
+| **spec.services.name** |    **NO**   | Represents a unique name of the service used by the Service Catalog. |
+| **spec.services.displayName** |    **YES**   | Specifies a human-readable name of the Remote Environment service. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.description** |    **NO**   | Provides a short, human-readable description of the service offered by the RE. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.longDescription** |    **NO**   | Provides a longer, human-readable description of the service offered by the RE. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.providerDisplayName** |    **YES**   | Specifies a human-readable name of the Remote Environment service provided. Parameter provided by the Metadata Service, do not edit. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/044-remote-evironment-custom-resource.md]: Specifies a human-readable name of the `provided/offered` Remote Environment service ~provided~.

File: docs/application-connector/docs/044-remote-evironment-custom-resource.md [hunk 2]
```diff
@@ -29,9 +29,27 @@ spec:
...
+| **spec.services.tags** |    **NO**   | Specifies additional tags used for better documentation of the available APIs. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.labels** |    **NO**   | Specifies additional labels for the service offered by the RE. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.entries** |    **YES**   | Contains the information about the APIs and Events that the service offered by the RE provides. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.entries.type** |    **YES**   | Specify the entry type: API or Event. Parameter provided by the Metadata Service, do not edit. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/044-remote-evironment-custom-resource.md]: `Specifies whether the entry is of API or Event type.`  ? -> for sure you should be consistent and use 3rd form of the verb at the beginning

File: docs/application-connector/docs/044-remote-evironment-custom-resource.md [hunk 3]
```diff
@@ -29,9 +29,27 @@ spec:
...
+| **spec.services.entries.gatewayUrl** |    **NO**   | Specifies the URL of the Application Connector. This field is required for the API entry type. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.entries.accessLabel** |    **NO**   | Specifies the label used in Istio rules in the Application Connector. This field is required for the API entry type. |
+| **spec.services.entries.targetUrl** |    **NO**   | Specifies the URL to a given API. This field is required for the API entry type. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.entries.oauthUrl** |    **NO**   | Specifies the URL used to authorize with a given API. This field is required for the API entry type. Parameter provided by the Metadata Service, do not edit. |
+| **spec.services.entries.credentialsSecretName** |    **NO**   | Specifies the name of the Secret which allows you to a call to a given API. This field is required if the **spec.services.entries.oauthUrl** is specified. Parameter provided by the Metadata Service, do not edit. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/044-remote-evironment-custom-resource.md]: Specifies the name of the Secret which allows you to ~a~ call ~to~ a given API. ?

**Comment by pkosiec** [comment]
@tomekpapiernik Next time please write PR name in imperative mood. All rules regarding commit messages apply there as well. The pull request name is needed for changelog generation and we have to be consistent to make it readable 🙂

**Pull request merged**